cmake/Modules/opm-material-prereqs.cmake

```diff
@@ -16,8 +16,8 @@ set (opm-material_DEPS
 	# compile with C++0x/11 support if available
 	"CXX11Features REQUIRED"
 	# prerequisite OPM modules
-	"opm-common"
 	"opm-parser"
+	"opm-common"
```

Comment:
The the order seems wrong to me. If opm-parser is listed first as not required then opm-common will be checked as not required implicitely. Listing it afterwards as REQUIRED will have no effect. A similar problem was fixed in OPM/opm-common#84